README.md

```diff
@@ -77,9 +77,20 @@ To deploy SMP server on [Linode](https://www.linode.com/):
 
 Please submit an [issue](https://github.com/simplex-chat/simplexmq/issues) if any problems occur.
 
-## 🚧 Deploy SMP server on DigitalOcean 🚧
+[<img alt="digitalocean" src="./img/digitalocean.svg" align="right" width="200">](https://cloud.digitalocean.com)
 
-Coming soon.
+## Deploy SMP server on DigitalOcean
+
+To deploy SMP server on [DigitalOcean](https://cloud.digitalocean.com):
+- Create a DigitalOcean account or login with an already existing one;
+- Click [create Droplet](https://cloud.digitalocean.com/droplets/new?size=s-1vcpu-1gb&region=fra1);
+- Choose the region and plan according to your requirements (for regular use cheapest Regular option should be sufficient).
```

Comment:
`(for regular use Regular option...)` sounds like a tautology, just `(Regular should be sufficient)`